tests/ui

```diff
@@ -0,0 +1,88 @@
+#!/bin/bash
+set -eux
+
+BROWSER="${1:-chromium}"
+
+echo "==> Install LXD"
+install_lxd
+
+echo "==> Setup node"
+if ! command -v node > /dev/null; then
+  snap install node --channel=20/stable --classic
+fi
+node --version
+
+echo "==> Get ui test code"
+install_deps git
+if [[ $LXD_SNAP_CHANNEL =~ ^5\.0 ]]; then
+  # The UI in LXD 5.0 is not compatible with the test suite in the main branch of lxd-ui,
+  # fall back to the old test suite from when LXD 5.0 was released.
+  git clone --depth=1 --single-branch --branch stable-5.0 https://github.com/canonical/lxd-ui.git
+else
+  git clone --depth=1 --single-branch --branch main https://github.com/canonical/lxd-ui.git
```

Comment:
@edlerd we should be checking out the test suite from the appropriate branch for the snap channel being tested, which includes 5.21 too, not just 5.0.

Reply:
The main branch test suite is compatible with 5.21 and 6.x branches. As we will extend coverage, I'd like to keep the suite from the main branch for both.

Reply:
We have tried this with lxd-ci and in the long run it becomes a challenge to maintain. Ultimately we plan to move much of lxd-ci into the main lxd repo's test suite.

As you already have a per-branch test suites I would encourage you to run the appropriate test suite for the branch, even if you end up backporting the same tests into both branches.

In the long run this will likely work out more maintainable as the branches diverge than having to put in lots of exceptions in the main test suite to cater for the old UI.